diff --git a/core/src/main/java/org/geonetwork/http/proxy/HttpProxyServlet.java b/core/src/main/java/org/geonetwork/http/proxy/HttpProxyServlet.java
index f8834ee..6033415 100644
--- a/core/src/main/java/org/geonetwork/http/proxy/HttpProxyServlet.java
+++ b/core/src/main/java/org/geonetwork/http/proxy/HttpProxyServlet.java
@@ -335,7 +335,13 @@ public class HttpProxyServlet extends HttpServlet {
                 PrintWriter out = response.getWriter();
                 String body = RequestUtil.inputStreamAsString(request);
 
-                final ContentType contentType1 = ContentType.create(request.getContentType(),
+                String ct = request.getContentType();
+                if (ct != null) {
+                	String[] cts = ct.split(";");
+                	ct = cts[0].trim();
+                }
+                
+                final ContentType contentType1 = ContentType.create(ct,
                         request.getCharacterEncoding());
                 StringEntity entity = new StringEntity(body, contentType1);
                 httpPost.setEntity(entity);
